docs/quest_template.md

```diff
@@ -251,347 +377,245 @@ Like all flag based fields, **QuestFlags** can be added for the different types
-entry = questC      PrevQuestId = questB     NextQuestId = 0       ExclusiveGroup = 0       NextQuestInChain = questD
-entry = questD      PrevQuestId = questC     NextQuestId = 0       ExclusiveGroup = 0       NextQuestInChain = 0
-```
+Text sent to player when the player tries to talk to the NPC with the quest active but incomplete. (The text under the "Progress" title in Wowhead.) You can use certain placeholders that will be filled in in-game: $B - line break, $N - name, $R - race, $C - class, $Gmale:female; (male and female can be replace with any synonymn you want, but the order must stay the same. IE: boy:girl / man:woman / sir:madam / dude:chick)
+
+### RequiredNpcOrGo1
+### RequiredNpcOrGo2
+### RequiredNpcOrGo3
+### RequiredNpcOrGo4
 
-### Chain of quests with multiple start quests.
+Value &gt; 0:required creature\_template ID the player needs to kill/cast on in order to complete the quest.
+Value &lt; 0:required gameobject\_template ID the player needs to cast on in order to complete the quest.
+If\*RequiredSpellCast\*is != 0, the objective is to cast on target, else kill.
+NOTE: If RequiredSpellCast is != 0 and the spell has effects Send Event or Quest Complete, this field may be left empty.
 
-Player should only be allowed to complete one of three possible
+### RequiredNpcOrGoCount1
+### RequiredNpcOrGoCount2
+### RequiredNpcOrGoCount3
+### RequiredNpcOrGoCount4
 
-```sql
-    questA     questB    questC
-      \           |         /
-        ------ questD -----
-                  |
-               questE
-```
+The number of times the creature or gameobject must be killed or casted upon.
 
-```sql
-entry = questA      PrevQuestId = 0        NextQuestId = questD    ExclusiveGroup = questA    NextQuestInChain = questD    
-entry = questB      PrevQuestId = 0        NextQuestId = questD    ExclusiveGroup = questA    NextQuestInChain = questD
-entry = questC      PrevQuestId = 0        NextQuestId = questD    ExclusiveGroup = questA    NextQuestInChain = questD
-entry = questD      PrevQuestId = 0        NextQuestId = 0         ExclusiveGroup = 0         NextQuestInChain = questE
-entry = questE      PrevQuestId = questD   NextQuestId = 0         ExclusiveGroup = 0         NextQuestInChain = 0
-```
+### RequiredItemId1
+### RequiredItemId2
+### RequiredItemId3
+### RequiredItemId4
+### RequiredItemId5
+### RequiredItemId6
 
-### Chain of quests with multiple start quests.
+[Id](item_template#id) of required item to complete the quest.
 
-Player must complete all three initial quests before D becomes available
+### RequiredItemCount1
+### RequiredItemCount2
+### RequiredItemCount3
+### RequiredItemCount4
+### RequiredItemCount5
+### RequiredItemCount6
 
-```sql
-    questA    questB    questC
-      \         |          /
-       ------ questD -----
-                |
-              questE
-```
+Amount of required items
+
+### Unknown0
```

Comment:
Is this a column?

Reply:
It seems so.

![colum](https://user-images.githubusercontent.com/2810187/122138767-cc2ff300-ce1d-11eb-9cf3-b662aa6a0f71.png)